content/en/blog/_posts/2025-xx-xx-jobs-managedby-goes-ga.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: "Kubernetes v1.35: Job Managed By Goes GA"
+date: 2025-XX-XX
+draft: true
+slug: kubernetes-v1-35-job-managedby-for-jobs-goes-ga
+author: >
+  [Dejan Zele Pejchev](https://github.com/dejanzele) (G-Research),
+  [Michał Woźniak](https://github.com/mimowo) (Google)
+---
+
+In Kubernetes v1.35, the managedBy field for Jobs graduated to General Availability (GA).
+
+This feature allows external controllers to take full responsibility for Job reconciliation, unlocking powerful scheduling patterns like multi-cluster dispatching with [MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/).
+
+## Why delegate Job reconciliation?
+
+The primary motivation for this feature is to support multi-cluster batch scheduling architectures, such as MultiKueue.
+
+Specifically in the MultiKueue design, we distinguish between a Management Cluster and a pool of Worker Clusters:
+- The Management Cluster is responsible for dispatching Jobs but not executing them. It needs to accept Job objects to track status, but it skips the creation and execution of Pods.
+- The Worker Clusters receive the dispatched work and execute the actual Pods.
+- Users interact only with the Management Cluster. Because the status is automatically propagated back, they can observe the Job's progress "live" without accessing the worker clusters.
+
+By using `spec.managedBy`, the MultiKueue controller on the Management Cluster can take over the reconciliation of a Job. It copies the status from the "mirror" Job running on the Worker Cluster back to the Management Cluster.
+
+Why not just disable the Job controller? While one could theoretically achieve this by disabling the built-in Job controller entirely, this is often impossible or impractical for two reasons:
+1. Managed Control Planes: In many cloud environments, the Kubernetes control plane is locked, and users cannot modify controller manager flags.
+2. Hybrid Workloads: Users often need a "hybrid" mode where the Management Cluster dispatches some heavy workloads to remote clusters but still executes smaller or control-plane-related Jobs locally. `spec.managedBy` allows this granularity on a per-Job basis.
+
+## How `spec.managedBy` works
```

Comment:
Additionally, shouldn't we mention status restrictions so that we can explicitly represent that this feature does not allow developers to get off the track of kube-controller-manager restrictions?

Reply:
I think this is very technical for the blogpost, where we take the approach to focus on the main use-case  (MultiKueue) rather than hypothetical controllers written from scratch (we haven't seen any yet). 

We have this [note](https://github.com/kubernetes/website/pull/53010/files#diff-0fc61ddf1a7c36ee7b7458abf41000f12c1112343624837d304abafd38704a88R50). Maybe we could extend it to mention at high-level something like:
```
(...)

If you are looking into implementing an external controller be aware that your controller needs to
be conformant with the definitions for the
[Job API](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/).
```
wdyt?

Reply:
I think the above suggestion would be useful to have. Even though we mention that the delegation is the main use case, we should set the right expectations in case people want to implement the full controller. The blog could be the first place people come across it.